### PR TITLE
add proper naming for every terminal

### PR DESCRIPTION
**Pull request by nanashi-1**
- Addresses: #64 

Not sure about some of the names.

### CODE REVIEW

File: nautilus_open_any_terminal/open_any_terminal_extension.py [hunk 1]
```diff
@@ -112,6 +112,36 @@
     "tabby": "-e",
 }
 
+TERM_NAME = {
+    "alacritty": "Alacritty",
+    "blackbox": "BlackBox",
+    "cool-retro-term": "cool-retro-term",
+    "deepin-terminal": "Deepin Terminal",
+    "foot": "Foot",
+    "footclient": "FOOTCLIENT",
```
Comment (Stunkymonkey) [inline nautilus_open_any_terminal/open_any_terminal_extension.py:121]: why is this all capital? Not sure this is wanted.

**Review by Stunkymonkey — COMMENTED** [review]
nice idea

File: nautilus_open_any_terminal/open_any_terminal_extension.py (hunk 1 above)
Comment (nanashi-1) [reply nautilus_open_any_terminal/open_any_terminal_extension.py:121]: Is FootClient good?

Comment (Stunkymonkey) [reply nautilus_open_any_terminal/open_any_terminal_extension.py:121]: I would prefer it yes. Having this all bold looks weird to me

Comment (nanashi-1) [reply nautilus_open_any_terminal/open_any_terminal_extension.py]: done 👍

**Pull request merged**